client/app/fileurl_darwin.go

```diff
@@ -1,6 +1,6 @@
 //go:build systray && darwin
```

Comment:
should remove the `systray` tag requirement from the darwin and windows fileurl sources